src/test/java/uk/gov/hmcts/ccd/domain/types/CaseDataValidatorTest.java

```diff
@@ -229,210 +229,6 @@ public void validValueCollection() throws Exception {
-            + "                    }, {\n"
-            + "                        \"code\": \"JUDGEDREDD\",\n"
-            + "                        \"label\": \"Judge Dredd\"\n"
-            + "                    }]\n"
-            + "                }\n"
-            + "            }\n"
-            + "        ],\n"
-            + "        \"DynamicRadioList\": {\n"
-            + "            \"value\": {\n"
-            + "                \"code\": \"JUDGESMITH\",\n"
-            + "                \"label\": \"Judge Smith\"\n"
-            + "            },\n"
-            + "            \"list_items\": [{\n"
-            + "                \"code\": \"JUDGEJUDY\",\n"
-            + "                \"label\": \"Judge Judy\"\n"
-            + "            }, {\n"
-            + "                \"code\": \"JUDGERINDER\",\n"
-            + "                \"label\": \"Judge Rinder\"\n"
-            + "            }, {\n"
-            + "                \"code\": \"JUDGESMITH\",\n"
-            + "                \"label\": \"Judge Smith\"\n"
-            + "            }, {\n"
-            + "                \"code\": \"JUDGEDREDD\",\n"
-            + "                \"label\": \"Judge Dredd\"\n"
-            + "            }]\n"
-            + "        },\n"
-            + "        \"DynamicMultiSelectList\": {\n"
-            + "            \"value\": [{\n"
-            + "                \"code\": \"MONDAYFIRSTOFMAY\",\n"
-            + "                \"label\": \"Monday, May 1st\"\n"
-            + "            }, {\n"
-            + "\n"
-            + "                \"code\": \"THURSDAYFOURTHOFMAY\",\n"
-            + "                \"label\": \"Thursday, May 4th\"\n"
-            + "            }],\n"
-            + "            \"list_items\": [{\n"
-            + "                \"code\": \"MONDAYFIRSTOFMAY\",\n"
-            + "                \"label\": \"Monday, May 1st\"\n"
-            + "            }, {\n"
-            + "                \"code\": \"TUESDAYSECONDOFMAY\",\n"
-            + "                \"label\": \"Tuesday, May 2nd\"\n"
-            + "            }, {\n"
-            + "                \"code\": \"WEDNESDAYTHIRDOFMAY\",\n"
-            + "                \"label\": \"Wednesday, May 3rd\"\n"
-            + "            }, {\n"
-            + "                \"code\": \"THURSDAYFOURTHOFMAY\",\n"
-            + "                \"label\": \"Thursday, May 4th\"\n"
-            + "            }]\n"
-            + "        }\n"
-            + "    }";
-
-        final Map<String, JsonNode> values = MAPPER.readValue(DATA, new TypeReference<HashMap<String, JsonNode>>() {
-        });
-        final ValidationContext validationContext = getValidationContextDynamicFields(values);
-        final List<ValidationResult> result = caseDataValidator.validate(validationContext);
-        assertEquals(result.toString(), 0, result.size());
-    }
-
```

Comment:
let's not forget to reenable this flaky test

Reply:
done